x-pack/legacy/plugins/apm/public/components/app/Settings/AddSettings/AddSettingFlyoutBody.tsx

```diff
@@ -10,17 +10,49 @@ import {
   EuiForm,
   EuiFormRow,
   EuiButton,
-  EuiFieldNumber,
+  EuiFieldText,
   EuiTitle,
   EuiSpacer,
   EuiHorizontalRule,
   EuiText
 } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
+import { isEmpty } from 'lodash';
 import { FETCH_STATUS } from '../../../../hooks/useFetcher';
 import { Config } from '../SettingsList';
 import { ENVIRONMENT_NOT_DEFINED } from '../../../../../common/environment_filter_values';
 
+const NO_SELECTION = 'NO_SELECTION';
+const EuiSelectWithPlaceholder: typeof EuiSelect = props => (
```

Comment:
Maybe a comment here that it's a workaround for the EUI/Chrome bug?

Reply:
+1. Comment and extract it to a separate file.